packages/lesswrong/lib/collections/dialogueChecks/schema.ts

```diff
@@ -22,21 +22,27 @@ const schema: SchemaType<DbDialogueCheck> = {
     ...schemaDefaultValue(false),
     canRead: ['members'],
     canCreate: ['members'],
-    canUpdate: [userOwns],
   },
   checkedAt: {
     type: Date,
     nullable: false,
     canRead: ['members'],
     canCreate: ['members'],
-    canUpdate: [userOwns],
   },
   match: {
     type: Boolean,
     nullable: false,
     canRead: ['members'],
     // Defined in server/resolvers/dialogueChecksResolvers.ts
   },
+  hideInRecommendations: {
+    type: Boolean,
+    nullable: false,
+    ...schemaDefaultValue(false),
+    canRead: ['members'],
+    canCreate: ['members'],
+    canUpdate: [userOwns],
```

Comment:
This should not have a `canUpdate` if it's being updated via a custom upsert mutation